examples/foreach/foreach.bal

```diff
@@ -7,24 +7,24 @@ public function main() {
     // The `foreach` statement can be used to iterate an array. Each iteration returns an element in the array. 
     // The index of the corresponding element is not returned.
     foreach var v in fruits {
-        io:println("fruit: " + v);
+        io:println("fruit: ", v);
     }
 
     io:println("\nIterating a map :");
     map<string> words = { a: "apple", b: "banana", c: "cherry" };
 
-    // Iterating a `map` will return the key (`string`) and the value as a `tuple` variable.
+    // Calling the `.entries()' method on a `map` will return the key (`string`) and the value as a `tuple` variable.
     // Tuple destructuring can be used to split the tuple variable in to two variables.
-    foreach var (k, v) in words {
-        io:println("letter: " + k + ", word: " + v);
+    foreach var [k, v] in words.entries() {
+        io:println("letter: ", k, ", word: ", v);
     }
 
     io:println("\nIterating a JSON object :");
     json apple = { name: "apple", colors: ["red", "green"], price: 5 };
-    // Iterating a JSON is not supported. To iterate a JSON, first convert the JSON to a `map` and then iterate
-    // the newly created `map`.
-    map<json> mapValue = <map<json>> map<json>.convert(apple);
-    foreach var (i, j) in mapValue {
+    // Iterating a JSON is only supported with `map<json>` or `json[]`.
+    // To iterate a JSON, first cast it to the relevant iterable type.
+    map<json> mapValue = <map<json>> apple;
+    foreach var [i, j] in mapValue.entries() {
```

Comment:
Same comment as above regarding the use of `entries()`. Only use `entries()` if you need the key as well.

Reply:
Changed as suggested.